tardis/montecarlo/tests/test_rpacket.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "../src/rpacket.h"
+
+int testing_rpacket_get_nu();
+
+int
+testing_rpacket_get_nu(){
+	rpacket_t * rp = (rpacket_t *) malloc(sizeof(rpacket_t));
+	double value = 10.2;
+	/*rpacket_set_nu(rp, value);
```

Comment:
I don't know. can you put the error that it is throwing here? maybe we can figure out why. (there will be a lot more C python stuff so it might be useful to learn these things).